sql/catalyst/src/main/scala/org/apache/spark/sql/Encoders.scala

```diff
@@ -135,6 +136,13 @@ object Encoders {
    */
   def BINARY: Encoder[Array[Byte]] = ExpressionEncoder()
 
+  /**
+   * An encoder for [[CalendarInterval]]
```

Comment:
For some reasons, looks we failed to link `CalendarInterval`. We can make it `` `CalendarInterval` ``, for now.

```
[error] /home/jenkins/workspace/SparkPullRequestBuilder/sql/catalyst/target/java/org/apache/spark/sql/Encoders.java:109: error: reference not found
[error]    * An encoder for {@link CalendarInterval}
[error]                            ^
```

Reply:
ah, thanks